indico/modules/events/surveys/fields/choices.py

```diff
@@ -136,3 +150,18 @@ def validators(self):
     def wtf_field_kwargs(self):
         return {'choices': [(x['id'], x['option']) for x in self.question.field_data['options']],
                 'coerce': lambda x: x}
+
+    def get_summary(self):
+        counter = Counter()
+        for answer in self.question.answers:
+            for choice in answer.data:
+                counter[choice] += 1
```

Comment:
`for answer in self.question.answers:
    counter.update(Counter(answer.data))
`

I am not sure, but maybe this is enough